source/Localization/en_US.xaml

```diff
@@ -119,6 +120,8 @@ If disabled, the game page will be opened to manually start the install process.
     <sys:String x:Key="LOCHumbleImportChoiceGamesTooltip">Import games available as part of Humble Choice subscription.</sys:String>
     <sys:String x:Key="LOCHumbleGeneralLibrary">Import Humble Library games</sys:String>
     <sys:String x:Key="LOCHumbleGeneralLibraryTooltip">Import standard Humble store purchases and bundle purchases.</sys:String>
+    <sys:String x:Key="LOCHumbleImportGameExtras">Also import game extras</sys:String>

```

Comment:
Unnecessary duplicate string, just define one generic one not tied to any plugin.